### DISCUSSION TITLE
Pasting Pictures

### DISCUSSION THREAD

**Discussion by brainbow1** [zim-desktop-wiki/zim-desktop-wiki#1665] [Q&A] [answered]
Hello!

I just experimented a bit with pasting pictures..
lets say I make a screenshot to clipboard, and then paste it in zim (or pasting any other picture into zim)..

- If I after a while decide to delete the picture from the text, and I look in the Folderstructure with all the .txt files, I see that it is still there. Is that behaviour changeable? I find it good not to delete things too early, but I imagine when I do that a few times, there are lots of old picture snippets that I deleted long ago, and they mix up with the actual pictures that I want to be there... is there a way to work around this? 

- is it possible to change the size of inserted pictures?

I would appreciate ur help!

**Comment by sojusnik** [comment, marked as answer]
>     * If I after a while decide to delete the picture from the text, and I look in the Folderstructure with all the .txt files, I see that it is still there. Is that behaviour changeable? I find it good not to delete things too early, but I imagine when I do that a few times, there are lots of old picture snippets that I deleted long ago, and they mix up with the actual pictures that I want to be there... is there a way to work around this?

See https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/1584

>     * is it possible to change the size of inserted pictures?

Right-click on the picture and choose edit picture.

**Reply by introt** [reply]
Changing picture sizes was broken in the `develop` branch for a second (#1664), so if you're using that, make sure to update!

**Reply by casperl** [reply]
Changing pictures works great but I will file a feature request to have a default picture size included in preferences since I have to go through the process of changing the picture size on most pictures I paste in order to make them fit on the screen.

**Reply by introt** [reply]
@casperl known issue, see #324. Someone should probs make a custom tool for this in the meantime

**Reply by sojusnik** [reply]
> Changing pictures works great but I will file a feature request to have a default picture size included in preferences since I have to go through the process of changing the picture size on most pictures I paste in order to make them fit on the screen.

+1 for having a default picture size in the settings! @casperl Can you kindly create a feature request for that under https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/categories/ideas?

**Reply by casperl** [reply]
https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/324 is open and this addresses my comment above.  I posted a suggestion into the comments that I hope will be helpful.
